python/ray/tests/test_task_events.py

```diff
@@ -0,0 +1,53 @@
+from collections import defaultdict
+from typing import Dict
+
+import ray
+from ray._private.test_utils import (
+    raw_metrics,
+    wait_for_condition,
+)
+from ray._private.worker import RayContext
+
+_SYSTEM_CONFIG = {
+    "task_events_report_interval_ms": 100,
+    "metrics_report_interval_ms": 200,
+}
+
+
+def count_by_type(info: RayContext) -> Dict:
+    res = raw_metrics(info)
+    task_events_info = defaultdict(int)
+    if "ray_gcs_task_manager_task_events_count" in res:
```

Comment:
nit: typically you'd use two separate metrics in this case, since COUNT vs STORED are not summable (don't add to 100%).